docs/finschia-tutorials/installation.md

```diff
@@ -24,9 +24,21 @@ source ~/.bash_profile
 **Go 1.20+** is required for the Finschia SDK.
 :::
 
+## Static build(Optional)
+
+### CentOS
+
+```bash
+# May need to install static libraries for linking
+$ sudo yum install glibc-static.x86_64 libstdc++-static -y
+
+# Build statically
+LINK_STATICALLY=true make build
+```
+
```

Comment:
I think it seems to need installing in CentOS, MacOS is OK. I think Ubuntu also will be ok. 
Please, add other OS case.

Reply:
Okay I'll check and update results.

Reply:
Tested on Ubuntu, CentOS, and MacOS.